src/mlpack/tests/init_rules_test.cpp

```diff
@@ -197,6 +198,56 @@ BOOST_AUTO_TEST_CASE(GaussianInitTest)
   BOOST_REQUIRE_EQUAL(weights3d.n_slices, slices);
 }
 
+/**
+ * Simple test of the VarianceScalingNormalInit class.
```

Comment:
Can you clarify a little in your comment about what we are testing in this test please?

Reply:
Okay. I will edit the test file in the next commit.

Reply:
Just a note, this one is still not handled yet.  I wanted to make sure it does not fall off your radar. :)